core/templates/pages/library-page/search-bar/search-bar.component.ts

```diff
@@ -89,7 +90,7 @@ export class SearchBarComponent implements OnInit, OnDestroy {
   }
 
   isMobileViewActive(): boolean {
-    return this.windowDimensionsService.getWidth() <= 766;
+    return this.windowDimensionsService.getWidth() <= 639;
```

Comment:
?

Reply:
In the Figma design, we are changing the search bar from desktop view to mobile view at 640px.

Reply:
Hi, @heyimShivam I find a new way that does not require to modify it.
Thanks!